spec/Body/Body.vspec

```diff
@@ -161,11 +161,55 @@ Windshield.WasherFluid.Level:
 ##
 Lights:
   type: branch
-  description: All lights.
+  instances: ["Beam", "Running", "Backup", "Parking", "Brake", "Fog", "Hazard", "Indicator"]
+  description: Exterior lights.
 
-# Include lights specification and attach it to the Lights branch
-#include ExteriorLights.vspec Lights
+Lights.Beam:
+  type: branch
+  instances: ["Low","High"]
+  description: Beam lights.
+#include StaticLights.vspec Lights.Beam
+
+Lights.Running:
+  type: branch
+  description: Running lights.
+#include StaticLights.vspec Lights.Running
+
+Lights.Backup:
+  type: branch
+  description: Backup lights.
+#include StaticLights.vspec Lights.Backup
+
+Lights.Parking:
+  type: branch
+  description: Parking lights.
+#include StaticLights.vspec Lights.Parking
+
+Lights.Fog:
+  type: branch
+  instances: ["Rear","Front"]
+  description: Fog lights.
+#include StaticLights.vspec Lights.Fog
+
+Lights.LicensePlate:
+  type: branch
+  description: License plate lights.
+#include StaticLights.vspec Lights.LicensePlate
+
+Lights.Brake:
+  type: branch
+#include BrakeLights.vspec Lights.Brake
+
+Lights.Hazard:
+  type: branch
+  description: Hazard lights.
+#include SignalingLights.vspec Lights.Hazard
 
+Lights.DirectionIndicator:
+  type: branch
+  instances: ["Left","Right"]
+  description: Indicator lights.
+#include SignalingLights.vspec Lights.Indicator
```

Comment:
Change `Lights.Indicator`to `Lights.DirectionIndicator`

Reply:
Thank you for that input!